webui/react/playwright.config.ts

```diff
@@ -23,7 +23,7 @@ export default defineConfig({
   fullyParallel: !!process.env.CI,
 
   /* https://playwright.dev/docs/test-timeouts#global-timeout */
-  globalTimeout: process.env.PWDEBUG ? 0 : 5 * 60 * 1000, // 3 min unless debugging
+  globalTimeout: process.env.PWDEBUG ? 0 : 500_000, // 500s unless debugging
```

Comment:
can we revisit this in the next PR now that devcluster is in and there aren't 1500 users?

Reply:
we also have another ticket to simply revisit all timeouts later. i'll remove the change for now